Cubical/Categories/Pullback.agda

```diff
@@ -13,32 +13,32 @@ private
   variable
     ℓ ℓ' : Level
 
-record Cospan (𝒞 : Precategory ℓ ℓ') : Type (ℓ-max ℓ ℓ') where
+record Cospan (C : Precategory ℓ ℓ') : Type (ℓ-max ℓ ℓ') where
   constructor cospan
   field
-    S₁ S₂ vertex : Precategory.ob 𝒞
-    s₁ : hom 𝒞 S₁ vertex
-    s₂ : hom 𝒞 S₂ vertex
+    S₁ S₂ vertex : Precategory.ob C
```

Comment:
Maybe using `l` and `r` (for left/right) would be better?

Reply:
done